packages/expo-modules-core/android/src/main/cpp/JavaCallback.cpp

```diff
@@ -58,47 +58,39 @@ jni::local_ref<JavaCallback::javaobject> JavaCallback::newInstance(
 }
 
 void JavaCallback::invoke() {
-  CallbackArg callbackArg((std::monostate()) );
-  callback(callbackArg);
+  // passing null
+  callback(folly::dynamic());
```

Comment:
ok, this is smart, I think I thought that would be equivalent to returning `{}` instead of `null` when thinking If I can do that.